comps/comps-katello-pulpcore-el8.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0"?>
+<!DOCTYPE comps PUBLIC "-//Red Hat, Inc.//DTD Comps info//EN" "comps.dtd">
+<comps>
+  <!--  <meta> -->
+  <!-- Meta information will go here eventually -->
+  <!--  </meta> -->
+  <group>
+    <id>katello-pulpcore</id>
+    <name>pulpcore for Katello</name>
+    <description>pulpcore Packages</description>
+    <uservisible>true</uservisible>
+    <packagelist>
+      <packagereq type="default">pulpcore-selinux</packagereq>
+      <packagereq type="default">python-aiohttp-debuginfo</packagereq>
```

Comment:
Why are these `python-` instead of `python3-` for all these `debuginfo` and `debugsource` packages?

Reply:
Probably a fair question to ask @evgeni since this file is just a copy of the EL7 comps minus the postgresql packages.

Reply:
I'm fine with merging and dealing with it later then

Reply:
And for what it's worth, from the mashing output:
```
20:40:40  2020-03-12 19:40:40 [WARNING ] In comps but not in katello-nightly/pulpcore/el7 tree: python-pycryptodomex-debugsource
20:40:40  2020-03-12 19:40:40 [WARNING ] In comps but not in katello-nightly/pulpcore/el7 tree: python-cryptography-debuginfo
20:40:40  2020-03-12 19:40:40 [WARNING ] In comps but not in katello-nightly/pulpcore/el7 tree: python-psycopg2-debuginfo
20:40:40  2020-03-12 19:40:40 [WARNING ] In comps but not in katello-nightly/pulpcore/el7 tree: python-ruamel-yaml-clib-debuginfo
20:40:40  2020-03-12 19:40:40 [WARNING ] In comps but not in katello-nightly/pulpcore/el7 tree: python-ruamel-yaml-clib-debugsource
20:40:40  2020-03-12 19:40:40 [WARNING ] In comps but not in katello-nightly/pulpcore/el7 tree: python-aiohttp-debuginfo
20:40:40  2020-03-12 19:40:40 [WARNING ] In comps but not in katello-nightly/pulpcore/el7 tree: python-cffi-debuginfo
20:40:40  2020-03-12 19:40:40 [WARNING ] In comps but not in katello-nightly/pulpcore/el7 tree: python-multidict-debugsource
20:40:40  2020-03-12 19:40:40 [WARNING ] In comps but not in katello-nightly/pulpcore/el7 tree: python-cffi-debugsource
20:40:40  2020-03-12 19:40:40 [WARNING ] In comps but not in katello-nightly/pulpcore/el7 tree: python-pycparser-debuginfo
20:40:40  2020-03-12 19:40:40 [WARNING ] In comps but not in katello-nightly/pulpcore/el7 tree: python-pycparser-debugsource
20:40:40  2020-03-12 19:40:40 [WARNING ] In comps but not in katello-nightly/pulpcore/el7 tree: python-markupsafe-debugsource
20:40:40  2020-03-12 19:40:40 [WARNING ] In comps but not in katello-nightly/pulpcore/el7 tree: python-yarl-debugsource
20:40:40  2020-03-12 19:40:40 [WARNING ] In comps but not in katello-nightly/pulpcore/el7 tree: python-multidict-debuginfo
20:40:40  2020-03-12 19:40:40 [WARNING ] In comps but not in katello-nightly/pulpcore/el7 tree: python-pyyaml-debugsource
20:40:40  2020-03-12 19:40:40 [WARNING ] In comps but not in katello-nightly/pulpcore/el7 tree: python-cryptography-debugsource
20:40:40  2020-03-12 19:40:40 [WARNING ] In comps but not in katello-nightly/pulpcore/el7 tree: python-pycryptodomex-debuginfo
20:40:40  2020-03-12 19:40:40 [WARNING ] In comps but not in katello-nightly/pulpcore/el7 tree: python-yarl-debuginfo
20:40:40  2020-03-12 19:40:40 [WARNING ] In comps but not in katello-nightly/pulpcore/el7 tree: python-pyyaml-debuginfo
20:40:40  2020-03-12 19:40:40 [WARNING ] In comps but not in katello-nightly/pulpcore/el7 tree: python-psycopg2-debugsource
20:40:40  2020-03-12 19:40:40 [WARNING ] In comps but not in katello-nightly/pulpcore/el7 tree: python-aiohttp-debugsource
20:40:40  2020-03-12 19:40:40 [WARNING ] In comps but not in katello-nightly/pulpcore/el7 tree: python-markupsafe-debuginfo
```

Reply:
https://github.com/theforeman/foreman-packaging/pull/4854